Crypto/Crypto/Presentation/CryptoApp.swift

```diff
@@ -11,7 +11,7 @@ import FirebaseCore
 
 @main
 struct YourApp: App {
-    @StateObject var contentViewModel = CoinListViewModel()
+    @StateObject var mainViewModel = MainViewModel()
```

Comment:
@OrlandoNicolasMarchioli, please remind me to return here, review the Dependency Injection pattern, and improve those ViewModel instantiations.